backend/submissions/models.py

```diff
@@ -0,0 +1,91 @@
+from django.core import exceptions
+from django.conf import settings
+from django.db import models
+
+from django.utils.translation import ugettext_lazy as _
+
+from model_utils.models import TimeStampedModel
+
+
+class Submission(TimeStampedModel):
+    conference = models.ForeignKey(
+        'conferences.Conference',
+        on_delete=models.CASCADE,
+        verbose_name=_('conference'),
+        related_name='submissions'
+    )
+
+    title = models.CharField(_('title'), max_length=100)
+    abstract = models.TextField(_('abstract'), max_length=1000)
+    elevator_pitch = models.TextField(_('elevator pitch'), max_length=300, default='', blank=True)
+    notes = models.TextField(_('notes'), default='', blank=True)
+
+    speaker = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        verbose_name=_('owner'),
+        on_delete=models.PROTECT,
+        related_name='submissions'
+    )
+
+    topic = models.ForeignKey(
+        'conferences.Topic',
+        verbose_name=_('topic'),
+        on_delete=models.PROTECT
+    )
+
+    language = models.ForeignKey(
+        'languages.Language',
+        verbose_name=_('language'),
+        on_delete=models.PROTECT
+    )
+
+    type = models.ForeignKey(
+        'submissions.SubmissionType',
+        verbose_name=_('type'),
+        on_delete=models.PROTECT
+    )
+
+    duration = models.ForeignKey(
+        'conferences.Duration',
+        verbose_name=_('duration'),
+        on_delete=models.PROTECT
+    )
+
+    def clean(self):
```

Comment:
👏

Reply:
Do you think we should move this logic to the `Form`?

Reply:
I'd keep here for now, can't think of any reasons why we should move it to the forms :)